### ISSUE TITLE
can not do a plugin for Magento\CustomAttributeManagement\Helper\Data

### ISSUE THREAD

**Issue by jorgealonso78** [magento/magento2#22775]
Preconditions (*)
php 7.1.26
magento 2.3.1
Steps to reproduce (*)
create a new module in this case (My_test)
 Declaring a plugin   
```
	<type name="Magento\CustomAttributeManagement\Helper\Data">
        <plugin name="My_test::HelperAttribute" type="My\test\Plugin\HelperAttribute" sortOrder="1"/>
    </type>
```
Defining a plugin
```
class HelperAttribute

{

    public function afterGetAttributeInputTypes(\Magento\CustomAttributeManagement\Helper\Data $subject, $result): array
    {
        $result['text']['validate_filters'][]      = 'regex';
        $result['multiline']['validate_filters'][] = 'regex';
        $result['date']['validate_filters'][]      = 'regex';
        return $result;
    }

    public function afterGetAttributeValidateFilters(\Magento\CustomAttributeManagement\Helper\Data $subject, $result): array
    {
        $result['regex'] = __('RetailONE Regular expression');
        return $result;
    }
}
```

Expected result (*)
have the new values on the result  for those functions  
Actual result (*)
get this when i go to  ..../admin/admin/customer_attribute/new/
 Attribute Information

    Properties Changes have been made to this section that have not been saved. This tab contains invalid data. Please resolve this before saving. 
![2019-05-07_11h42_10](https://user-images.githubusercontent.com/8143525/57315184-15e77480-70c1-11e9-88be-93a1b446849a.png)

**Comment by jorgealonso78** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @jorgealonso78. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @jorgealonso78, here is your [Magento instance](https://i-22775-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-22775-2-3-develop.instances.magento-community.engineering/admin](https://i-22775-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by hirenkpatel0** [comment]
Hello I am working on this in Magento 2.3-develop.

**Comment by hirenkpatel0** [comment]
I am working on this. Assign me in this

**Comment by jorgealonso78** [comment]
![2019-05-07_16h58_45](https://user-images.githubusercontent.com/8143525/57332769-6c68a900-70e9-11e9-934f-a6e23c13e46a.png)
[my.zip](https://github.com/magento/magento2/files/3154545/my.zip)

**Comment by jorgealonso78** [comment]
How long do you have to wait to review something that only takes 30 minutes? Can another person take this please

**Comment by jorgealonso78** [comment]
please this looks like a pinpon game assigned to one person assigned to another and the bug where same. 22 days to test only if it's a bug with all the things that is needed

**Comment by AlexWorking** [comment]
Sorry for delay @jorgealonso78. But as You can see from the above (pull request "Changed access for $allowLengthValidationTypes #24" has been created) the progress on Your issue hasn't stopped. Now the pull request needs to be verified. After that will have been done, it will be merged into 2.3-develop version. We'll try to do our best in order to fulfill all the procedures as quickly as possible. Thanks for Your patience in advance.

**Comment by engcom-Delta** [comment]
Hi @jorgealonso78 thank you for your report. I am not able to reproduce issue by steps you described on  
2.3-develop. 
Seems like issue was fixed by commit:
- [7704d1d17366a34ad024bdec382127e430aa1948](https://github.com/magento/partners-magento2ee/commit/7704d1d17366a34ad024bdec382127e430aa1948)
